python/ray/serve/tests/test_util.py

```diff
@@ -665,6 +667,21 @@ def test_get_all_live_placement_group_names(ray_instance):
     assert set(get_all_live_placement_group_names()) == {"pg3", "pg4", "pg5", "pg6"}
 
 
+def test_metrics_pusher():
```

Comment:
Without this change, this test almost never passes. `counter["val"]` at the end of the test is 16-17.